packages/core/content-type-builder/admin/src/pages/ListView/ListView.tsx

```diff
@@ -6,7 +6,7 @@ import has from 'lodash/has';
 import isEqual from 'lodash/isEqual';
 import upperFirst from 'lodash/upperFirst';
 import { useIntl } from 'react-intl';
-import { Prompt, useRouteMatch } from 'react-router-dom';
+import { unstable_usePrompt as usePrompt, useMatch } from 'react-router-dom';
```

Comment:
same comment as earlier about usePrompt

Reply:
Same response – OOS for the PR, I would talk to DX about the handling the CTB in this case :)